docs/docs/tuning-your-model.mdx

```diff
@@ -434,9 +434,48 @@ underlying tensorflow ops `tf.sparse.sparse_dense_matmul`,
 `tf.nn.sparse_softmax_cross_entropy_with_logits`,
 and `tf.math.unsorted_segment` ops. For more information see [here](https://github.com/tensorflow/community/blob/master/rfcs/20210119-determinism.md)
 
-You can see which featurizers are sparse [here](./components.mdx#featurizers),
+For the above reasons, the models are also not guaranteed to yield the exact same
+performance when trained on a GPU across multiple runs. This even applies to situations where training is run
+multiple times on the same training data with the same config
+and random seeds set appropriately while being evaluated on a standard held-out test set.
+Internal experiments have shown the following fluctuation in model's performance when trained and evaluated on a held-out
+test set of a variety of datasets (experiments were run 5 times on each dataset):
+
+```
++-----------------------+------------+------------------+------------------+-------------------+
+| Task,                 | Number of  | Average standard | Minimum standard | Maximum standard  |
+| Metric                | datasets   | deviation        | deviation        | deviation         |
++-----------------------+------------+------------------+------------------+-------------------+
+| Intent Classification | 11         | 0.0042           | 2.4e-5           | 0.0176            |
```

Comment:
Can you give the average here as well? I assume this means sth like `0.87 (+/- 0.042)` and not `87 (+/- 0.042)`. I think making that obvious would be good.

Reply:
The average standard deviation is across 11 (or the corresponding number of datasets available for the task) datasets, where the actual F1 score would be different for each dataset. So you can't really write something like `0.87 (+/- 0.0042)`

I could mention the range of the metric in the `Task, Metric` column itself, if you think that's helpful.

Reply:
I think that will help, yes.

Reply:
Added